tests/e2e/aws_e2e.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""End to end test for the aws module."""
+import json
+import os
+import unittest
+import logging
+
+from libcloudforensics import aws
+
+log = logging.getLogger()
+EC2_SERVICE = 'ec2'
+
+
+class EndToEndTest(unittest.TestCase):
+  """End to end test on AWS.
+
+  This end-to-end test runs directly on AWS and tests that:
+    1. The aws.py module connects to the target instance and makes a snapshot
+        of the boot volume (by default) or of the volume passed in parameter
+        to the aws.CreateVolumeCopy() method.
+    2. A new volume is created from the taken snapshot.
+
+  To run this test, add your project information to a project_info.json file:
+
+  {
+    "instance": "xxx", # required
+    "zone": "xxx" # required
+    "volume_id": "xxx", # optional
+  }
+
+  Export a PROJECT_INFO environment variable with the absolute path to your
+  file: "user@terminal:~$ export PROJECT_INFO='absolute/path/project_info.json'"
+  """
```

Comment:
What about the credentials needed to access the AWS account?

Reply:
Ah yeah, I added a link to [AWS credentials doc](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html) which is what people should configure.